ats/management.py

```diff
@@ -68,7 +62,7 @@ def restart(self):
         AtsTest.restart()
 
     def filter (self, *filters):
-        "Add filters to the list. Clear list if no arguments."
+        "Add filters to the list. Clear list if no argum    ents."
```

Comment:
inadvertent spacing

Reply:
Hello v-hemish, are you working with someone else for these ATS changes.  Just wondering what the motivation is.